episodes/15-explanation.md

```diff
@@ -105,34 +133,79 @@ How might you prioritise what to keep if you have to cut it down?
-  - Header hierarchy (no h1 headers in the lesson body, no skipped levels e.g. h2 -> h4)
-  - Use descriptive link text - no "click here" or "this page" etc
-  - Image contrast \& tools to check this
+## Other Important Considerations for Lesson Text
+
+When writing your lesson text you also want to avoid unintentionally demotivating your learners.
+Make time to review your text for:
+
+- [dismissive language][dismissive-language-instructor-training] - e.g. 'simply', 'just'
+- use of [stereotypes][stereotype-threat-instructor-training] (check learner profiles for stereotypes too)
+- [expert awareness gaps][expert-awareness-gap-instructor-training], i.e. places where you may be assuming the learners know more than they actually do
+- fluid representations, i.e. using different terms with the same meaning interchangeably
+- unexplained or unnecessary jargon 
+- unexplained assumptions
+- sudden jumps in difficulty/complexity
+
+You should also review your text thinking about accessibility.
+This includes:
+
+  - Avoiding regional/cultural references and idioms that will not translate across borders/cultures
+  - Avoiding contractions i.e. don't, can't, won't etc
+  - Checking that all figures/images have well written alternative text, including [writing altnerative text for data visualizations](https://medium.com/nightingale/writing-alt-text-for-data-visualization-2a218ef43f81) - see [the Workbench documentation for directions on how to add alt-text and captions to images in a lesson](https://carpentries.github.io/sandpaper-docs/episodes.html#figures)
+  - Checking the header hierarchy - no h1 headers in the lesson body, no skipped levels
+  - [Using descriptive link text](https://www.imperial.ac.uk/staff/tools-and-reference/web-guide/training-and-events/materials/accessibility/links/) - no "click here" or "this page", etc
+  - [Checking the text and foreground contrast for images](https://contrastchecker.com/)
 
 ::::::::::::::::::::::::::::::::::::::  challenge
 
-## Exercise: Alternative text for images (5 minutes)
+## Optional Exercise: Alternative Text for Images (5 minutes)
+
+Which of the following is a good alt-text option for the image below?
+
+![Atmospheric Carbon Dioxide at Mauna Loa Observatory](https://gml.noaa.gov/webdata/ccgg/trends/co2_data_mlo.png){alt="Line graph of increasing carbon dioxide at the Mauna Loa Observatory from 1958 to present"}
+
+1. Graph of data
+2. Graph with increasing lines
+2. Line graph of increasing carbon dioxide in ppm at the Mauna Loa Observatory from 1958 to present
+3. Line graph of increasing carbon dioxide in ppm at the Mauna Loa Observatory, Hawaii, United States, from 1959 to present including values from each year. Red line shows variation in each year and black line is average for each year. 1959 = 315.90 ppm, 1960 = 316.91, 1961 = 317.64 ...
+
+_[Data/Image provided by NOAA Global Monitoring Laboratory, Boulder, Colorado, USA] (https://esrl.noaa.gov/)_
```

Comment:
Hmm.. I'm happy to commit but there is still another issue here.  Even when I don't include any italics, this link doesn't render in my local preview.
<img width="971" alt="Screen Shot 2022-05-31 at 9 51 06 AM" src="https://user-images.githubusercontent.com/5558419/171203344-7ab51007-4343-42b6-9373-8cc983de8e9c.png">

Could be my version of sandpaper (which usually the case) or maybe a bug?

Reply:
Ahh!  that was it lol... the line wrapping tricked me!

Reply:
Oh fun! I will add an issue to {pegboard} to catch this kind of link formatting. It will be a bit challenging because commonmark thinks that this is normal text and formats the link in the parens, but I should be able to find a way around it.